x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/support/SecuritySystemIndices.java

```diff
@@ -238,6 +238,10 @@ private XContentBuilder getMainIndexMappings() {
                             builder.startObject("allow_restricted_indices");
                             builder.field("type", "boolean");
                             builder.endObject();
+
+                            builder.startObject("remote_clusters");
```

Comment:
Note to self: double-check if there are any implications for cluster upgrades because of the mapping change or not